members/SOHL33351_720319.yaml

```diff
@@ -0,0 +1,115 @@
+bioguide: SOHL33351_720319
+contact_form:
+  method: post
+  action: ""
+  steps:
+    - visit: "http://www.ohiohouse.gov/jason-stephens/contact"
+    - fill_in:
+        - name: tbxFirstName
```

Comment:
Minor: the `name` for these steps should generally be a more human readable version of the value getting entered in case it ever needed to be served up. None of these would ever be seen so its nbd in this case